src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1882,6 +1882,94 @@ internal override IList<PSModuleInfo> ImportModulesUsingWinCompat(IEnumerable<st
         {
             IList<PSModuleInfo> moduleProxyList = new List<PSModuleInfo>();
 #if !UNIX
+            // the ModuleDeny list is cached in PowerShellConfig object
+            string[] moduleDenyList = System.Management.Automation.Configuration.PowerShellConfig.Instance.GetWindowsPowerShellCompatibilityModuleDenyList();
```

Comment:
The whole namespace is necessary?

Reply:
Added `using System.Management.Automation.Configuration;` and removed namespace here.